sdks/python/apache_beam/io/gcp/bigquery_tools.py

```diff
@@ -352,7 +352,12 @@ class BigQueryWrapper(object):
 
   HISTOGRAM_METRIC_LOGGER = MetricLogger()
 
-  def __init__(self, client=None, temp_dataset_id=None, temp_table_ref=None):
+  def __init__(
+      self,
+      client=None,
+      temp_dataset_id=None,
+      temp_table_ref=None,
+      project_id=None):
```

Comment:
Do we do anything with this project_id?

Reply:
From what I remember, running the code without assigning project_id resulted in the following error:
OSError: Project ID was not passed and could not be determined from the environment.

Reply:
I don't think it is actually getting assigned, though, and other functions take in project_id as parameters. If it is passed in to initialize the object, I think it will be ignored as written

Reply:
Oh I see. Should I go ahead and remove it, then?

Reply:
Yeah, lets go ahead and get rid of it - thanks